benchmark/Cpp/Savina/src/concurrency/LogisticMap.lf

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (C) 2020 TU Dresden
+ * 
```

Comment:
The comments you wrote for the other benchmarks are great and super useful! They are missing from this benchmark.

Reply:
Right, this one slipped through. I honestly also don't understand well what exactly it is doing. I'll give it another look